bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -835,10 +790,8 @@ export default {
       if (window) {
         window.addEventListener('keydown', this.keyDown)
       }
-
-      if (localStorage.getItem('openhab.ui:webaudio.enable') === 'enabled') {
-        this.startEventSource()
-      }
+      // sse-event mixin
```

Comment:
Can you please remove the comment from this line, but keep the line? 
IMO makes the code cleaner, and if you hover over the method call, your IDE can anyway tell you the method is from the mixin (at least JetBrains IDEs do that).